### ISSUE TITLE
NDI Source not showing up in source menu in 4.6.2

### ISSUE THREAD

**Issue by itswoshi** [Palakis/obs-ndi#333]
I reinstalled NDI on my streaming computer and I use OBS 23.2.1 (The latest release) . I downloaded your hotfix, but I still do not see the Source plugin. ( I reinstalled OBS + NDI Plugin *tried the installer and the zip* )

**Comment by itswoshi** [comment]
[2019-08-24 07-41-44.txt](https://github.com/Palakis/obs-ndi/files/3537305/2019-08-24.07-41-44.txt)

**Comment by Palakis** [comment]
@itswoshi obs-ndi 4.6.2 got released a few hours ago and fixes this issue

**Comment by itswoshi** [comment]
@Palakis Thats what i downloaded..

**Comment by itswoshi** [comment]
![462](https://user-images.githubusercontent.com/38363692/63639061-5228eb80-c644-11e9-922b-95ec504f6833.png)


Provided screenshot so you can see its 4.6.2

**Comment by ahmedsohail405** [comment]
I'm facing the same problem.

**Comment by itswoshi** [comment]
@ahmedsohail405 upload your log file so he has more information on it.

**Comment by ahmedsohail405** [comment]
[2019-08-24 20-09-57.txt](https://github.com/Palakis/obs-ndi/files/3537322/2019-08-24.20-09-57.txt)

**Comment by Palakis** [comment]
@itswoshi Sorry for the misunderstanding, thanks for reporting this. Looks like I screwed up somewhere when releasing 4.6.2.

**Comment by guilhermecunha13** [comment]
Same problem here. I reinstalled NDI (latest-latest release) on my PC running OBS 23.2.1 still not showing up in source menu. Below my log file. Thanks in advance
===============================================
12:48:26.428: CPU Name: Intel(R) Core(TM) i3 CPU         540  @ 3.07GHz
12:48:26.429: CPU Speed: 3066MHz
12:48:26.429: Physical Cores: 2, Logical Cores: 4
12:48:26.429: Physical Memory: 3959MB Total, 3090MB Free
12:48:26.429: Windows Version: 6.1 Build 7601 (revision: 24511; 64-bit)
12:48:26.429: Running as administrator: false
12:48:26.429: Aero is Enabled
12:48:26.456: Current Date/Time: 2019-08-24, 12:48:26
12:48:26.456: Browser Hardware Acceleration: false
12:48:26.456: Portable mode: false
12:48:30.683: OBS 23.2.1 (64-bit, windows)
12:48:30.683: ---------------------------------
12:48:30.747: ---------------------------------
12:48:30.747: audio settings reset:
12:48:30.747: 	samples per sec: 44100
12:48:30.747: 	speakers:        2
12:48:30.816: ---------------------------------
12:48:30.816: Initializing D3D11...
12:48:30.817: Available Video Adapters: 
12:48:30.819: 	Adapter 0: Intel(R) HD Graphics
12:48:30.819: 	  Dedicated VRAM: 67108864
12:48:30.819: 	  Shared VRAM:    1711276032
12:48:30.819: 	  output 0: pos={0, 0}, size={1920, 1080}, attached=true
12:48:30.819: 	  output 1: pos={1920, 0}, size={1360, 768}, attached=true
12:48:30.961: Loading up D3D11 on adapter Intel(R) HD Graphics (0)
12:48:30.977: D3D11 loaded successfully, feature level used: 40960
12:48:33.603: ---------------------------------
12:48:33.603: video settings reset:
12:48:33.603: 	base resolution:   1920x1080
12:48:33.603: 	output resolution: 1280x720
12:48:33.603: 	downscale filter:  Bicubic
12:48:33.603: 	fps:               30/1
12:48:33.603: 	format:            NV12
12:48:33.603: 	YUV mode:          601/Partial
12:48:33.604: NV12 texture support not available
12:48:33.615: Audio monitoring device:
12:48:33.615: 	name: Default
12:48:33.615: 	id: default
12:48:33.629: ---------------------------------
12:48:33.799: [CoreAudio encoder]: CoreAudio AAC encoder not installed on the system or couldn't be loaded
12:48:33.878: Failed to load 'en-US' text for module: 'decklink-ouput-ui.dll'
12:48:34.159: [AMF] AMF Test failed due to one or more errors.
12:48:34.159: Failed to initialize module 'enc-amf.dll'
12:48:35.344: Module '../../obs-plugins/64bit/libEGL.dll' not loaded
12:48:35.654: [obs-browser]: Version 2.4.0
12:48:37.878: [obs-ndi] hello ! (version 4.6.2)
12:48:37.878: [obs-ndi] Trying 'C:\Program Files\NewTek\NewTek NDI 3.8 Runtime\v3'
12:48:37.887: [obs-ndi] Found NDI library at 'C:/Program Files/NewTek/NewTek NDI 3.8 Runtime/v3/Processing.NDI.Lib.x64.dll'
12:48:39.118: [obs-ndi] NDI runtime loaded successfully
12:48:39.121: [obs-ndi] NDI library initialized successfully (NDI SDK WIN64 18:40:40 May 10 2019 3.8.2.0)
12:48:39.169: obs_register_source: Tried to register obs_source_info with size 288 which is more than libobs currently supports (280)
12:48:39.169: obs_register_source: Tried to register obs_source_info with size 288 which is more than libobs currently supports (280)
12:48:39.169: obs_register_source: Tried to register obs_source_info with size 288 which is more than libobs currently supports (280)
12:48:39.169: obs_register_source: Tried to register obs_source_info with size 288 which is more than libobs currently supports (280)
12:48:47.751: Couldn't find VLC installation, VLC video source disabled
12:48:48.110: No blackmagic support
12:48:49.494: ---------------------------------
12:48:49.494:   Loaded Modules:
12:48:49.494:     win-wasapi.dll
12:48:49.494:     win-mf.dll
12:48:49.494:     win-ivcam.dll
12:48:49.494:     win-dshow.dll
12:48:49.494:     win-decklink.dll
12:48:49.494:     win-capture.dll
12:48:49.494:     vlc-video.dll
12:48:49.494:     text-freetype2.dll
12:48:49.494:     rtmp-services.dll
12:48:49.494:     obs-x264.dll
12:48:49.494:     obs-vst.dll
12:48:49.494:     obs-transitions.dll
12:48:49.494:     obs-text.dll
12:48:49.494:     obs-qsv11.dll
12:48:49.494:     obs-outputs.dll
12:48:49.494:     obs-ndi.dll
12:48:49.494:     obs-filters.dll
12:48:49.494:     obs-ffmpeg.dll
12:48:49.494:     obs-browser.dll
12:48:49.494:     image-source.dll
12:48:49.494:     frontend-tools.dll
12:48:49.494:     enc-amf.dll
12:48:49.494:     decklink-ouput-ui.dll
12:48:49.494:     coreaudio-encoder.dll
12:48:49.494: ---------------------------------
12:48:49.588: ==== Startup complete ===============================================
12:48:49.735: Service '' not found
12:48:50.007: Switched to Preview/Program mode
12:48:50.007: ------------------------------------------------
12:48:50.219: All scene data cleared
12:48:50.219: ------------------------------------------------
12:48:50.645: WASAPI: Device 'LG TV (Intel(R) Display Audio)' initialized
12:48:50.775: WASAPI: Device 'Desktop Microphone (Cinema - Microsoft LifeCam.)' initialized
12:48:50.842: Switched to scene 'Scene'
12:48:50.849: ------------------------------------------------
12:48:50.849: Loaded scenes:
12:48:50.849: - scene 'Scene':
12:48:50.849: ------------------------------------------------
12:48:51.103: adding 441 milliseconds of audio buffering, total audio buffering is now 441 milliseconds (source: Desktop Audio)
12:48:51.103: 
12:48:51.616: [obs-ndi] starting NDI main output with name 'WinCam'
12:48:51.791: [obs-ndi] 'WinCam': ndi output started
12:48:51.791: [obs-ndi] starting NDI preview output with name 'OBS Preview'
12:48:51.798: [obs-ndi] 'OBS Preview': ndi output started
